source/fuzz/fuzzer_pass_interchange_signedness_of_integer_operands.cpp

```diff
@@ -91,6 +91,13 @@ void FuzzerPassInterchangeSignednessOfIntegerOperands::Apply() {
 
 uint32_t FuzzerPassInterchangeSignednessOfIntegerOperands::
     FindOrCreateToggledIntegerConstant(uint32_t id) {
+  // |id| must represent a non-specialization constant because the constant
+  // manager does not declare specialization constants.
```

Comment:
Sorry for missing this before but I think the more accurate comment is:

|id| must not be a specialization constant because we do not know the value of specialization constants.

Reply:
Even if the constant manager could handle them, I don't think we could use them here.

Reply:
Done.